app/models/mixins/scanning_operations_mixin.rb

```diff
@@ -29,27 +29,6 @@ def save_metadata_op(xmlFile, type, jobid = nil)
     true
   end
 
-  def agent_job_message_op(jobid, message)
-    _log.info "jobid: [#{jobid}] starting"
-    begin
-      Timeout.timeout(WS_TIMEOUT) do
-        MiqQueue.put(
-          :class_name  => "Job",
-          :method_name => "agent_message_update_queue",
-          :args        => [jobid, message],
-          :task_id     => "agent_job_message_#{Time.now.to_i}",
-          :zone        => MiqServer.my_zone,
-          :role        => "smartstate"
-        )
-        return true
-      end
-    rescue Exception => err
-      _log.log_backtrace(err)
-      ScanningOperations.reconnect_to_db
-      return false
```

Comment:
This is probably around remote agents running this code and talking back to MangaeIQ, which is no longer needed.